pkg/testframework/framework.go

```diff
@@ -39,8 +44,8 @@ func DeleteAll(t *testing.T, client *Clientset) {
 	}
 }
 
-func getResource(client *Clientset, resource string) (metav1.Object, error) {
-	var res metav1.Object
+func GetResource(client *Clientset, resource string) (runtime.Object, error) {
```

Comment:
`runtime.Object` too generic?  I think we lose type safety if we use these.  Necessary?

Reply:
yeah its too generic and we dont need all of the interface features. Was speaking with @mfojtik and he advised me to use runtime.Object, since its more generic and there is a smaller chance you get an object that we cant handle